charts/redis-ha/templates/_configs.tpl

```diff
@@ -31,6 +31,9 @@
 {{- if .Values.auth }}
     sentinel auth-pass {{ template "redis-ha.masterGroupName" . }} replace-default-auth
 {{- end }}
+{{- if .Values.sentinel.auth }}
+    requirepass replace-default-auth
```

Comment:
Isn't this going to mean the password is always statically `replace-default-auth`? 

I think we should add the ability for users to define the password, or choose an existing password from an existing secret if we're adding this functionality. 

I may be misunderstanding though, please let me know.

Reply:
I guess on line 135, replace-default-auth is being replaced by the actual password in sentinel.conf
```
 sed -i "s/replace-default-auth/${ESCAPED_AUTH}/" "$REDIS_CONF" "$SENTINEL_CONF"
```

Having said that it seems like the $ESCAPED_AUTH variable that "replace-default-auth" is being replaced with, is still the same as redis password which means redis and sentinel password will be same.

If I can, I would suggest looking at https://github.com/DandyDeveloper/charts/pull/37
specifically at file: "charts/redis-ha/templates/_configs.tpl", it's pretty much same thing:

```
{{- if .Values.sentinel.auth }}
    requirepass replace-default-sentinel-auth
{{- end }}
```

but, "replace-default-sentinel-auth" is replaced with environment variable "$SENTINELAUTH", which basically comes from secret (but this secret is different from the one used for redis password)

```
if [ "${SENTINELAUTH:-}" ]; then
        echo "Setting sentinel auth values"
        ESCAPED_AUTH_SENTINEL=$(echo "$SENTINELAUTH" | sed -e 's/[\/&]/\\&/g');
        sed -i "s/replace-default-sentinel-auth/${ESCAPED_AUTH_SENTINEL}/" "$SENTINEL_CONF"
    fi
```
This way one can define seperate secrets from redis and sentinel passwords

Hope this helps

Reply:
@SarjakSShah Would you say it's worthwhile closing this in favor of that larger PR? I'll be jumping on that as soon as I can, possibly later in my afternoon.

Reply:
Yes this PR just makes the sentinel password the same as the redis password instead of no password.
It would be nice to be able to set the sentinel password in the secret file as well just like you would set the redis password but I have not had time to do that.

Reply:
> @SarjakSShah Would you say it's worthwhile closing this in favor of that larger PR? I'll be jumping on that as soon as I can, possibly later in my afternoon.

I believe so